common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
Body clipping on the neck. It may be fixed with your rigging.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120006273-cba7f780-bfe9-11eb-9486-5b3c9839f3fa.png)

</details>